src/flight-search/filtering.js

```diff
@@ -5,6 +5,19 @@ function filterByPrice(trip, priceRange) {
   return trip.fares[0] && utils.filterByRange(trip.fares[0].price.amountUsd, priceRange);
 }
 
+function filterByFlexibleTickets(trip, flexible = []) {
+  const isRefundableFlag = flexible.map(str => str.toLowerCase()).indexOf("refundable") !== -1;
```

Comment:
can we use Array.includes() here instead?

Reply:
updated